jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -195,10 +209,23 @@ public void testBuild_simple() throws IOException, InterruptedException, DigestE
               "No classes files were found - did you compile your project?"));
     }
 
+    String output = JibRunHelper.buildAndRun(simpleTestProject, targetImage);
+
     Assert.assertEquals(
         "Hello, world. An argument.\n1970-01-01T00:00:01Z\nrw-r--r--\nrw-r--r--\nfoo\ncat\n"
             + "1970-01-01T00:00:01Z\n1970-01-01T00:00:01Z\n",
-        JibRunHelper.buildAndRun(simpleTestProject, targetImage));
+        output);
+
+    String digest =
+        assertDigestFile(simpleTestProject.getProjectRoot().resolve("build/jib-image.digest"));
+    String imageReferenceWithDigest = ImageReference.parse(targetImage).withTag(digest).toString();
+    Assert.assertEquals(
+        output, JibRunHelper.buildAndRun(simpleTestProject, imageReferenceWithDigest));
```

Comment:
Do we have to run this again? `jib-image.id` is also generated when pushing to a registry. I'd like to avoid adding another Jib run if it's not so meaningful.

Reply:
Not sure what you mean; this is run twice to make sure the digest generated by Jib is valid. The same logic happens in the maven integration tests, it's just been missing from gradle for a while it seems.

Reply:
Ah... let me look at this again.